lib/configure.js

```diff
@@ -280,6 +280,43 @@ function configure (gyp, argv, callback) {
       }
     }
 
+    // For z/OS we need to set up the path to zoslib include directory,
+    // which contains headers included in v8config.h.
+    var zoslibIncDir
+    if (process.platform === 'os390') {
+      logprefix = "find zoslib's zos-base.h:"
+      let msg
+      var zoslibIncTmp = process.env.ZOSLIB_INCLUDES
```

Comment:
The variable name `zoslibIncTmp` makes me think that the path is in tmpdir. Maybe a better name could be `zoslibEnvar`, `zoslibPath`, or something else?